lib/chat/user_message_rows.dart

```diff
@@ -47,8 +47,8 @@ const dummyData = [
   },
 ];
 
-class Messages extends StatelessWidget {
-  Messages({Key key}) : super(key: key);
+class UserMessageRows extends StatelessWidget {
```

Comment:
same here (its not letting me hightlight 😭 ), you can also place "Messages" in a constant file